superbench/config/default.yaml

```diff
@@ -4,6 +4,10 @@ superbench:
   benchmarks:
     matmul:
       enable: true
+      modes:
+        - name: local
+          proc_num: 8
+          prefix: NVIDIA_VISIBLE_DEVICES={proc_rank}
```

Comment:
Shall we change to CUDA_VISIBLE_DEVICES?

Reply:
updated